rules/DeadCode/Rector/ClassMethod/RemoveDelegatingParentCallRector.php

```diff
@@ -133,6 +133,9 @@ private function isMethodReturnType(ClassMethod $classMethod, string $type): boo
         return $this->isName($classMethod->returnType, $type);
     }
 
+    /**
+     * @return Expr|StaticCall|null
```

Comment:
the `Expr` seems need to be excluded, as it always return only `StaticCall` or `null`